stdlib/2and3/locale.pyi

```diff
@@ -110,3 +110,4 @@ def str(float: float) -> _str: ...
 
 locale_alias: Dict[_str, _str]  # undocumented
 locale_encoding_alias: Dict[_str, _str]  # undocumented
+windows_locale: Dict[int, _str]
```

Comment:
I believe that like the previous two variables, this should have an `# undocumented` comment.

Reply:
Hi.

Thanks for the reply.

I will add the comments and resubmit.

Cheers.

SekouD.